### PR TITLE
Added docker file to run bower.github.io as container

### PR DESCRIPTION
**Pull request by urodoz**

### CODE REVIEW

**Comment by riyadhalnur** [comment]
@bower/contributors thoughts?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM node:5.10.0
```
Comment (paulohp) [inline Dockerfile]: Hey @urodoz do you know alpine linux? This one here looks pretty solid: https://github.com/mhart/alpine-node

**Comment by paulohp** [comment]
I really liked this :)

I just did some comments on the code @urodoz.

Thanks.

File: Dockerfile (hunk 1 above)
Comment (urodoz) [inline Dockerfile]: I usually work on top of official Docker images, but if it's needed i can change the base image on the commit to **mhart/alpine-node**

**Pull request merged**